packages/node_modules/@webex/plugin-meetings/src/reconnection-manager/index.js

```diff
@@ -281,32 +313,40 @@ export default class ReconnectionManager {
       throw (new NeedsRetryError(syncError));
     }
 
+    // If the meeting state was left, no longer reconnect media
     if (this.meeting.state === _LEFT_) {
-      try {
-        rejoinMeeting = true;
-
-        await this.meeting.join();
-      }
-      catch (joinError) {
-        this.rejoinAttempts += 1;
-        if (this.rejoinAttempts <= this.maxRejoinAttempts) {
-          LoggerProxy.logger.info('ReconnectionManager:index#executeReconnection --> Unable to rejoin meeting, reconnecting.', joinError);
-          throw (new NeedsRetryError(joinError));
-        }
-        else {
-          LoggerProxy.logger.error('ReconnectionManager:index#executeReconnection --> Unable to rejoin meeting after max attempts.', joinError);
-          this.status = RECONNECTION.STATE.FAILURE;
-          throw joinError;
-        }
-      }
+      throw (new NeedsRejoinError({wasSharing}));
```

Comment:
nice so it would throw an error and rejoin but it wont be the same as reconnect flow

Reply:
Correct, as soon as we know we have to rejoin, our "reconnect" has failed and we go into the rejoin flow.